### ISSUE TITLE
Media mapper error after updating Media Type

### ISSUE THREAD

**Issue by aar-nemlig** [umbraco/Umbraco.Heartcore.Issues#166]
### Summary

After updating an Image Media Type to add an extra field called "AccessibilityText", the Log viewer got full of errors indicating a failure while mapping the media.

<img width="1455" height="761" alt="Image" src="https://github.com/user-attachments/assets/b4916226-5a44-45fa-967f-1ec966c5099d" />
<img width="1420" height="700" alt="Image" src="https://github.com/user-attachments/assets/eca3b7b3-9b3a-4604-98a5-18b1aa6f429f" />

I think this is related to this ticket: [Issue](https://github.com/umbraco/Umbraco.Heartcore.Issues/issues/160)

### Steps to reproduce

1. Have a project setup with media data content in place, images and videos
2. Go into Settings -> Media Types-> Image
3. Add a property called "AcessibilityText"
4. Add a Textbox to that property
5. Press save
6. Check the Log Viewer for the errors.

### Expected result

To have just the field added to the Media Type

### Actual result

Throws 100's of errors in a second trying to remap the media

**Comment by m-tacheva** [comment]
Hi, any progress on this @jmf-umbraco ?

**Comment by jmf-umbraco** [comment]
Hi @m-tacheva,

This issue is likely to be a corrupt index or cache file in your project. We have not been able to reproduce it on a fresh project with a bunch of media.

For issues like this your best bet is to raise a support ticket through the Umbraco Cloud portal.

**Comment by jmf-umbraco** [comment]
Since this is likely a problem with this one specific project, I will close this issue. If you have further related challenges, please let us know through support.
